openapi/data_repository_service.swagger.yaml

```diff
@@ -396,3 +406,33 @@ definitions:
       license:
         type: object
         description: License information for the exposed API
+  BundleObjects:
```

Comment:
I suggest `BundleObject` (singular), since it's only one thing.

Reply:
Updated.